packages/commonwealth/server/routes/feed.ts

```diff
@@ -0,0 +1,136 @@
+import { AppError } from '@hicommonwealth/core';
+import { Thread, ThreadAttributes, type DB } from '@hicommonwealth/model';
+import { Feed } from 'feed';
+import { slugify } from 'utils';
+import { GetBulkThreadsResult } from '../controllers/server_threads_methods/get_bulk_threads';
+import { ServerControllers } from '../routing/router';
+import { TypedRequestQuery, TypedResponse } from '../types';
+import { formatErrorPretty } from '../util/errorFormat';
+import {
+  ActiveThreadsRequestQuery,
+  BulkThreadsRequestQuery,
+  GetThreadsRequestQuery,
+  GetThreadsResponse,
+  SearchThreadsRequestQuery,
+} from './threads/get_threads_handler';
+
+function toDate(t: ThreadAttributes): Date {
+  return t.last_edited ?? t.created_at;
+}
+
+function sorter(a: ThreadAttributes, b: ThreadAttributes) {
+  return toDate(a).getTime() - toDate(b).getTime();
+}
+
+function computeUpdated(bulkThreads: GetBulkThreadsResult): Date {
+  if (bulkThreads.threads.length === 0) {
+    // there are no threads
+    return new Date();
+  }
+
+  const sorted = [...bulkThreads.threads].sort(sorter);
+
+  // return the most recent thread and get its date
+  return toDate(sorted.reverse()[0]);
```

Comment:
We can just change the sorter function to sort in reverse instead of sorting it then reversing:
`toDate(a).getTime() - toDate(b).getTime();`

Reply:
I don't mind changing this because it's more performant but I generally don't do this because it's a 'feature' that sort of hides itself and can cause bugs.  

Good catch though.